### ISSUE TITLE
ftruncate analogue

### ISSUE THREAD

**Issue by userqq** [amphp/file#32]
Hi.
Is there any plans to add ftruncate() analogue or any workaround?
Now I'm using something like that
```        
yield \Amp\ParallelFunctions\parallel(function () use ($filename) {
    $fh = fopen($filename, 'c');
    ftruncate($fh, 1024);
    fclose($fh);
})();
```
but I think it's terrible solution

**Comment by trowski** [comment]
I'll add this function in the next release.

**Comment by userqq** [comment]
Great news.
Thank you

**Comment by trowski** [comment]
This has been implemented in master and will be part of v1.0.
